modules/ossm-enabling-strict-mtls-whole-service-mesh.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// install/ossm-enabling-mtls.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-enabling-strict-mtls-whole-service-mesh_{context}"]
+= Enabling strict mTLS across the whole service mesh
+
+You can configure mTLS across the entire mesh by applying the `PeerAuthentication` policy to the `istiod` namespace, such as `istio-system`.
```

Comment:
Maybe we should add that the `istiod` namespace equals what you set in the `spec.namespace` of your `Istio` resource?